source/agora/consensus/protocol/Nominator.d

```diff
@@ -211,9 +212,31 @@ extern(D):
 
     ***************************************************************************/
 
-    public bool receiveEnvelope (SCPEnvelope envelope) @trusted
+    public bool receiveEnvelope (SCPEnvelope envelope) @safe nothrow
```

Comment:
This attribute change and the trusted lambda call to `receiveEnvelope` should be part of its own commit.